src/components/Clock.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component } from 'react';
+
+interface Props {
+  clockName: string;
+}
+
+type ClockState = {
+  today: Date,
+};
+
+export class Clock extends Component<Props, ClockState> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({
+        today: new Date(),
+      });
+      console.info(this.state.today.toUTCString().slice(-12, -4));
```

Comment:
You can use this string, because linter not happy
// eslint-disable-next-line no-console